src/models/queryAggregate.js

```diff
@@ -0,0 +1,62 @@
+import QueryRangeGroupBy from './queryRangeGroupBy';
+import map from 'lodash.map';
+
+class QueryAggregate { // eslint-disable-line no-unused-vars
+
+    constructor(name) {
+      this.name = name;
+      this._aggregate = this._aggregate || {};
```

Comment:
do not introduce a aggregate level just this.groupBys (camelcased) (this.metrics)